repo2docker/buildpacks/conda/install-miniforge.bash

```diff
@@ -19,27 +18,27 @@ unset HOME
 wget --quiet $URL -O ${INSTALLER_PATH}
 chmod +x ${INSTALLER_PATH}
 
-# check md5 checksum
-if ! echo "${MD5SUM}  ${INSTALLER_PATH}" | md5sum  --quiet -c -; then
+# check sha256 checksum
+if ! echo "${SHA256SUM}  ${INSTALLER_PATH}" | sha256sum  --quiet -c -; then
     echo "md5sum mismatch for ${INSTALLER_PATH}, exiting!"
     exit 1
 fi
 
 bash ${INSTALLER_PATH} -b -p ${CONDA_DIR}
 export PATH="${CONDA_DIR}/bin:$PATH"
 
-# Allow easy direct installs from conda forge
-conda config --system --add channels conda-forge
```

Comment:
I think this is no longer needed, but someone should verify

Reply:
You could verify with something like:
```bash
conda config --show channels | grep conda-forge
```

Additionally, it might be worth trying:
```bash
conda config --set channel_priority true
```

as this will make solves both faster and more predictable

Reply:
We discussed the `channel_priority` setting previously and concluded that we want it set to "flexible" (which is done [here](https://github.com/jupyter/repo2docker/blob/8d490cf9d80f963f3746ab2bd04b9fb183b9bab9/repo2docker/buildpacks/conda/install-miniconda.bash#L46-L47))

Discussion https://github.com/jupyter/repo2docker/pull/719#issuecomment-507592182 and https://github.com/jupyter/repo2docker/issues/731

Reply:
Note that we don't have `defaults` added in `miniforge`. So you might want to add `defaults` as a lower priority channel here.

Reply:
Hmm interesting. I forgot about that

Reply:
Does this mean if I just have an `environment.yml` file now in my repo, It will pick from conda-forge only rather than defaults? Was this the previous behavior?

Reply:
If the `environment.yml` didn't explicitly have defaults, then yes, `defaults` will not be used. Previous behaviour was `conda-forge` + `defaults` which is different from the new bahaviour which is only `conda-forge`. You can make it go back to the previous behaviour with the suggestion I made earlier.

Reply:
@isuruf done!